pkg/resource/sensitive.go

```diff
@@ -212,7 +213,9 @@ func GetSensitiveParameters(ctx context.Context, client SecretClient, from runti
 					sensitives := make(map[string]interface{})
 					for key, value := range *sel {
 						sensitive, err = client.GetSecretValue(ctx, value)
-						if err != nil {
+						if err != nil && kerrors.IsNotFound(err) {
+							sensitive = []byte("")
```

Comment:
A note on why we need this would be helpful for further readers.

Reply:
done